modules/seqexec/server/src/main/scala/seqexec/server/ghost/GHOSTController.scala

```diff
@@ -22,10 +22,71 @@ import squants.time.Time
 final case class GHOSTController[F[_]: Sync](ghostClient: GHOSTClient[F],
                                              gdsClient: GDSClient) {
   import GHOSTController._
-  val log: Logger = getLogger
+  val Log: Logger = getLogger
+
+  // TODO: Are GIAPI parameters maintained between applies? Here is a use case that requires more information:
+  // 1. We observe in standard resolution mode using both IFUs on targets.
+  // 2. We observe in high resolution mode using only IFU1 on a target.
+  // Question: Do we need to, for (2), touch IFU2 at all (i.e. to standard mode, no target)?
+  // Do we need to always set target type for IFUs? How about bundle configuration?
+  //
+  // Right now, this only generate a config if the name, RA, and dec are defined: otherwise, we get the empty config.
+  private def ifuConfig(ifuNum: IFUNum,
+                        nameOpt: Option[String],
+                        raOpt: Option[HourAngle],
+                        decOpt: Option[Angle],
+                        bundleConfig: BundleConfig): Configuration = {
+    def cfg[P: Show](paramName: String, paramVal: P) =
```

Comment:
Do we use `Show` in this method?

Reply:
I think this maybe taken from `GPIController`. I used `Show` there but I think it is a bad idea. we should instead create another typeclass decoupling giapi formatting from regular formatting. I'll add a Jira task about it

Reply:
Yes, it is taken from `GPIController`, and won't compile unless `P` has `Show`. I wasn't too sure why that was the case, so that's probably a good idea.